quizcon/main/templatetags/quiz_tools.py

```diff
@@ -153,15 +153,18 @@ def right_answers_per_quiz(id):
         questions.update({total_right_answers(question): question})
```

Comment:
This tally approach only records a single question for a given number of right answers, e.g. if Question 1 has 2 correct answers and Question 2 has 2 correct answers, the questions dictionary will look like this after the forloop at Ln. 149 is complete.

`{2: <Question: Question object (2)>}`

The iteration of the questions dictionary at Ln 154 *is* attempting to account for multiple questions with the same score but will not succeed as the data will not be present from the first iteration.

The same issue exists for the functions `questions_most_incorrect` and `questions_most_idk`. 

For this analytics page, I wonder if a better / more interesting view would be to list each question, and then have columns for # correct, # idk & #incorrect?

Reply:
Ah I see it. Yes originally I was only looking for one question per section but realized that there may be more than one question that meets the same criteria. Thinking back I don't think having this section account for more than one question was really discussed (Mike's original plan was one). I like the idea of having a questions table.